lolstaticdata/items/__main__.py

```diff
@@ -118,7 +119,95 @@ def rewrite():
+        shutil.rmtree(os.path.join(directory, "__wiki__"))
+
+    if not os.path.exists(os.path.join(directory, "__wiki__")):
+        os.mkdir(os.path.join(directory, "__wiki__"))
+    # ddragon = DragonItem.get_json_ddragon()
+    cdragon = DragonItem.get_cdragon()
+    wikiItems = get_item_urls(False)
+
+    # print(wikiItems)
+
+    for i in cdragon:
+        i["name"] = i["name"].replace("%i:ornnIcon% ", "")
+        i["name"] = i["name"].replace("<rarityLegendary>", "")
+        if "</rarityLegendary>" in i["name"]:
+            i["name"] = i["name"].split("</rarityLegendary>")[0]
+    jsons = {}
+    for name,data in wikiItems.items():
+        if name in ["goose", "goose1"]:
+            continue
+        else:
+            item = None
+            print(name)
+            # try:
+            #     cdragon_item = DragonItem.get_item_cdragon(d, ddragon)
+            # except ValueError:
+            #     continue
+            if name == "'Your Cut'":
+                name = "Your Cut"
+                # l = [d for d in cdragon if x.upper() == d["name"].upper()]
+                l = list(filter(lambda d: d["name"].upper() == name.upper(), cdragon))
+                name = "'Your Cut'"
+            elif name == "Slightly Magical Boots":
+                if list(filter(lambda d: d["name"].upper() == name.upper(), cdragon)):
+                    l = list(filter(lambda d: d["name"].upper() == name.upper(), cdragon))
+                else:
+                    name = "Slightly Magical Footwear"
+                    # l = [d for d in cdragon if x.upper() == d["name"].upper()]
+                    l = list(filter(lambda d: d["name"].upper() == name.upper(), cdragon))
+                    name = "Slightly Magical Boots"
+            else:
+                # l = [d for d in cdragon if x.upper() == d["name"].upper()]
+                l = list(filter(lambda d: d["name"].upper() == name.upper(), cdragon))
+
+            if len(l) >= 1:
+                for i in l:
+
+                    cdrag_item = DragonItem.get_item_cdragon(i)
+                    wiki_item = WikiItem._parse_item_data(data,name,wikiItems)
+                    item = wiki_item
+                    item.icon = cdrag_item.icon
+                    item.id = int(cdrag_item.id)
+                    item.builds_from = cdrag_item.builds_from
+                    item.builds_into = cdrag_item.builds_into
+                    item.simple_description = cdrag_item.simple_description
+                    item.required_ally = cdrag_item.required_ally
+                    item.required_champion = cdrag_item.required_champion
+                    item.shop.purchasable = cdrag_item.shop.purchasable
+                    item.special_recipe = cdrag_item.special_recipe
+                    if item.iconOverlay == True:
+                        item.iconOverlay = (
+                            "http://raw.communitydragon.org/10.24/game/data/items/icons2d/bordertreatmentornn.png"
```

Comment:
Let's not hardcode old patch border urls.

Reply:
Merged.